manim/mobject/text/tex_mobject.py

```diff
@@ -381,6 +381,29 @@ def set_color_by_tex(self, tex, color, **kwargs):
             part.set_color(color)
         return self
 
+    def set_opacity_by_tex(
+        self, tex: str, opacity: float = 0.5, remaining_opacity: float = None, **kwargs
+    ):
+        """
+        Sets the opacity of the tex specified. If 'remaining_opacity' is specified,
```

Comment:
Just for consistency, and formatting

```suggestion
        """Sets the opacity of the tex specified. If ``remaining_opacity`` is specified,
```